examples/WORKSPACE

```diff
@@ -1,24 +1,21 @@
 workspace(name = "examples")
 
-load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
 
-http_archive(
+git_repository(
     name = "io_bazel_rules_rust",
-    sha256 = "a62597502c1ee14fcc30da92fc31c69bc6b436f464939449dc7be88c76218a31",
-    strip_prefix = "rules_rust-6e5fa2c570ac2f17ac1df840d060fc8aab521a07",
-    urls = [
-        # Master branch as of 2020-06-06
-        "https://github.com/bazelbuild/rules_rust/archive/6e5fa2c570ac2f17ac1df840d060fc8aab521a07.tar.gz",
-    ],
+    commit = "fda9a1ce6482973adfda022cadbfa6b300e269c3",
```

Comment:
I'd also add a comment that this is the master branch as of 2020-06-30

Reply:
Added